examples/tic-tac-toe/ui/src/config.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { createNetworkConfig } from '@mysten/dapp-kit';
+import { getFullnodeUrl } from '@mysten/sui/client';
+
+import LocalnetPackage from './env.localnet.ts';
+
+const NoPackage = { packageId: null, upgradeCap: null };
+
+const { networkConfig, useNetworkVariable } = createNetworkConfig({
```

Comment:
Looking at this, should we create a super simple parser for these configs looking for `.env.{network}.json` which will automatically read these? So if someone is calling has first called `./publish.sh`, they won't need to come here and import themselves.

Reply:
I can stub these out as files, instead -- that might be the easiest way.

Reply:
I got this to mostly work -- I stubbed out `env.localnet.ts` and locally I'm using `--skip-worktree` locally to avoid updating it -- not great, but it'll do for now.